contact.css

```diff
@@ -0,0 +1,253 @@
+  display: flex;
+  position: relative;
+}
+section .container .contactinfo .sci li {
+  list-style: none;
+  margin-right: 15px;
+}
+section .container .contactinfo .sci li a {
+  text-decoration: none;
+}
+section .container .contactinfo .sci li a img {
+  width: 30px;
+  filter: invert(1);
+  opacity: 0.5;
+}
+section .container .contactinfo .sci li:hover a img {
+  opacity: 1;
+}
+section .container .contactForm {
+  position: absolute;
+  padding: 70px 50px;
+  background: white;
+  margin-left: 150px;
+  padding-left: 250px;
+  width: calc(100% - 150px);
+  height: 100%;
+  box-shadow: 0px 50px 50px rgb(0, 0, 0, 0.5);
+}
+section .container .contactForm h2 {
+  font-weight: 300;
+  font-size: 24px;
+  color: #0f3959;
+}
+section .container .contactForm .formBox {
+  position: relative;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  padding-top: 30px;
+}
+section .container .contactForm .formBox .inputBox {
+  position: relative;
+  margin-bottom: 35px;
+}
+section .container .contactForm .formBox .inputBox.w50 {
+  width: 47%;
+}
+section .container .contactForm .formBox .inputBox.w100 {
+  width: 100%;
+}
+section .container .contactForm .formBox .inputBox input {
+  width: 100%;
+  padding: 5px 0;
+}
+section .container .contactForm .formBox .inputBox input,
+section .container .contactForm .formBox .inputBox textarea {
+  width: 100% !important;
+  color: #333;
+  resize: none;
+  font-size: 18px;
+  font-weight: 300;
+  padding: 5px;
+  border: none;
+  border-bottom: 1px solid #777;
+  outline: none;
+}
+section .container .contactForm .formBox .inputBox textarea {
+  min-height: 120px;
+}
+section .container .contactForm .formBox .inputBox span {
+  position: absolute;
+  left: 0;
+  font-size: 18px;
+  font-weight: 300;
+  padding: 5px;
+  color: #333;
+  transition: 0.5s;
+  pointer-events: none;
+  opacity: .5;
+}
+section .container .contactForm .formBox .inputBox textarea:focus ~ span,
+section .container .contactForm .formBox .inputBox input:focus ~ span,
+section .container .contactForm .formBox .inputBox textarea:valid ~ span,
+section .container .contactForm .formBox .inputBox input:valid ~ span {
+  transform: translateY(-20px);
+  font-size: 12px;
+  font-weight: 300;
+  color: #ff568c;
+  letter-spacing: 1px;
+  opacity: 1;
+}
+section .container .contactForm .formBox .inputBox input[type="submit"] {
+  position: relative;
+  cursor: pointer;
+  background: #0f3959;
+  color: white;
+  max-width: 150px;
+  padding: 12px;
+  border: none;
+}
+section .container .contactForm .formBox .inputBox input[type="submit"]:hover {
+  background: #ff568c;
+}
+
+/* responsive design starts here */
+@media (max-width: 1200px) {
+  section .container {
+    width: 90%;
+    min-width: auto;
+    margin: 20px;
+    box-shadow: 0 20px 50px rgb(0, 0, 0, 0.2);
+  }
+  section .container .contactinfo {
+    position: relative;
+    top: 0;
+    height: 550px;
+    /* box-shadow: none; */
```

Comment:
Why commented this?

Reply:
yes, for the small screen 
I want to remove the box-shadow 
that's why made changes in @media

Reply:
so is it in use of final output?

Reply:
nope